hr_expense_line_supplier/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+HR Expense Line Supplier
+========================
+
+This module adds a supplier field on the expense line.
+
+A second module will automatically install after this one, setting the field
+as 'required'. We need to separate this in 2 modules because we need the demo
+expense lines to have a supplier before setting the field required. This is
+done in order to avoid warnings on installation with demo data activated.
```

Comment:
@JulieSFL I just checked, there is no error, just a warning. An the warning is due to the fact that the not null constraint can not be applied. Seperating the feature in 2 modules does not help.